extensions/amp-embedly-card/1.0/web-component.js

```diff
@@ -8,16 +10,13 @@ import {
  * Registers `<bento-embedly-card> component to CustomElements registry
  */
 export function defineElement() {
```

Comment:
fmi: do you know why `defineElement` here defines both the card and the key, whereas `defineElementEmbedlyKey` only the key? in that case, why are they both exported

Reply:
> Is that true
Basically, yes

So this was necessary to allow consumers to do this:

```html
<head>
  <script async type="module" src="bento-embedly-card.js"/>
</head>
<body>
      <amp-embedly-key value={apiKey}></amp-embedly-key>
      <amp-embedly-card data-url="https://www.youtube.com/watch?v=lBTCB7yLs8Y"></amp-embedly-card>
</body>
```

the `bento-foo.js` scripts all register the "relevant" web-component(s) to the customElement registry by calling an autogenerated `defineElement` fn. When there are multiple relevant components for a particular extension, we would like them to all be registered. In those cases, we create `web-component.js` files, which override the autogenerated files built during the extension's build process, that implements a custom `defineElement` fn that registers all the necessary components.

Registering all the components together is basically what the amp components do already ([example](https://github.com/ampproject/amphtml/blob/main/extensions/amp-embedly-card/1.0/amp-embedly-card.js#L38-L41))

Although I guess it's up for debate, whether we should be exporting `defineElementEmbedlyKey` since it doesn't really make sense to use it on its own..

Reply:
> Although I guess it's up for debate, whether we should be exporting defineElementEmbedlyKey since it doesn't really make sense to use it on its own.

Please remove the export then if it doesn't make sense on its own

Reply:
Although, if this was preexisting feel free to punt to a future cleanup!

Reply:
adding to checklist [here](https://github.com/ampproject/amphtml/pull/37309#issuecomment-1011222406)